Signal/src/util/DeviceSleepManager.swift

```diff
@@ -71,7 +71,7 @@ import Foundation
 
         if UIApplication.shared.isIdleTimerDisabled != shouldBlock {
             if shouldBlock {
-                Logger.info("\(self.TAG) \(#function): Blocking sleep")
```

Comment:
Sure, just bear in mind that:

* there could be multiple block objects.
* For robustness, `DeviceSleepManager` uses weak references but we still need to be very careful to remove block objects ASAP.  Weak references are only a half-measure since their expiration can't be observed.